README.md

```diff
@@ -6,16 +6,59 @@
 [![sdk](https://img.shields.io/pypi/v/onepanel-sdk?color=01579b&label=sdk)](https://pypi.org/project/onepanel-sdk/)
 [![docs](https://img.shields.io/github/v/release/onepanelio/core?color=01579b&label=docs)](https://docs.onepanel.io)
 [![issues](https://img.shields.io/github/issues-raw/onepanelio/core?color=01579b&label=issues)](https://github.com/onepanelio/core/issues)
-[![chat](https://img.shields.io/badge/support-slack-01579b)](https://join.slack.com/t/onepanel-ce/shared_invite/zt-eyjnwec0-nLaHhjif9Y~gA05KuX6AUg)
+[![chat](https://img.shields.io/badge/support-slack-01579b)](https://onepanel-ce.slack.com/join/shared_invite/zt-eyjnwec0-nLaHhjif9Y~gA05KuX6AUg#/)
 [![license](https://img.shields.io/github/license/onepanelio/core?color=01579b)](https://opensource.org/licenses/Apache-2.0)
 
-Full stack vision AI platform with fully integrated modules for model building, semi-automated labeling, training and data pipelines, powered by Kubernetes.
+Production scale end-to-end vision AI platform with fully integrated modules for model building, automated labeling, training and data pipelines on Kubernetes.
+
+## Why Onepanel?
+
+- End-to-end workflow and infrastructure automation for production scale vision AI
+- Easily scale your data processing and training pipelines to multiple machines
+- Unified web interface and SDKs for all your deep learning tools and workflows
+- Scalability, flexiblity and resiliency of Kubernetes without the complexity
```

Comment:
Maybe expand?
"without the complexity of configuring kubernetes to support these workflows"